src/react/features/lenses/timeline/hooks/getTimelineData.js

```diff
@@ -41,9 +41,9 @@ const getDioryTimelineData = ({ diory = {}, diorys = [], parent }) => {
   const { lng } = getDioryDateLongitude({ diory, diorys, parent })
   return {
     id: diory.id,
-    center: lng && {
```

Comment:
useTimelineBounds has default location that requires center to be undefined if there is no location.
  DEFAULT_LOCATION: {
    lat: 0,
    lng: getDateLongitude({ date: new Date() }),
  },
However, this has a bug. It should have new Date().toISOString().
Maybe that fixes this problem.